scripts/feature_tests/README.md

```diff
@@ -19,25 +20,42 @@ It is recommended to run test-framework CI especially when
 introducing a commit related to pivoting/remediation/upgrade, to ensure that new
 changes will not break the existing functionalities.
 
-Test-framework can be triggered by leaving `/test-features` comment for
-remediation/pivoting and `/test-upgrade-features` comment for upgrade on a
-pull request.
+Test-framework can be triggered by leaving `/test-features` or `/test-features-centos`
```

Comment:
Maybe you could also add a note that /test-features is Ubuntu based, and /test-features -centos is Centos?

Reply:
Added, thanks